velox/exec/Operator.h

```diff
@@ -124,6 +124,12 @@ struct OperatorStats {
 
   CpuWallTiming finishTiming;
 
+  // Metric for operators to report CPU time spent on background activities
+  // (activities that are not running on driver threads). Responsibility of
+  // individual operators to report background CPU time at a reasonable time
```

Comment:
> Responsibility of

use active voice: "Operators are responsible for ..." + period at the end of the sentence.